README.md

```diff
@@ -1,20 +1,68 @@
-# eslint-config-10up
+# 10up Eslint Config
 
 > ESLint [shareable config](https://github.com/10up/eslint-config)
 
 [![Support Level](https://img.shields.io/badge/support-active-green.svg)](#support-level)
 
-[eslint-config-10up](https://github.com/10up/eslint-config) is a shareable configuration package for [eslint](http://eslint.org).
+[@10up/eslint-config](https://github.com/10up/eslint-config) is a shareable configuration package for [eslint](http://eslint.org) built on top of [eslint-airbnb-config](https://github.com/airbnb/javascript) and modified to meet 10up's own standards.
 
 ## Installation
 
-To use this config, install [@10up/eslint-config](https://github.com/10up/eslint-config) as a development dependecy of your project:
+1. Install all peer dependencies
+
+```sh
+npx install-peerdeps --dev @10up/eslint-config@next
+```
+
+2. Install [@10up/eslint-config](https://github.com/10up/eslint-config) as a development dependecy of your project:
 
 ```sh
 npm install @10up/eslint-config --save-dev
 ```
 
-Next, add this configuration to your `package.json`:
+## Avaliable ESLint configs
+
+### @10up/eslint-config
+
+The default export and contains common rules that are not specific to any framework or environment.
+
+### @10up/eslint-config/react
+
+Extends `@10up/eslint-config` adding specific rules to react.
+
+This rule requires installing `babel-eslint`
+
+```sh
+npm install --save-dev 'babel-eslint
+```
+
+### @10up/eslint-config/gutenberg
+
+Extends `@10up/eslint-config/react` adding the custom gutenberg rules from `@wordpress/eslint-plugin`.
+
+This rule requires installing the `@wordpress/eslint-plugin` package.
+
+```sh
+npm install --save-dev @wordpress/eslint-plugin
+```
+
+### @10up/eslint-config/node
+
+Extends `@10up/eslint-config` adding specific rules to Node.js.
+
+### @10up/eslint-config/jest
+
+Adds specic rules for the `jest` testing framework.
+
+This rule requires installing `eslint-plugin-jest`
+
+```sh
+npm install --save-dev eslint-plugin-jest
+```
+
+## Usage
+
+In order to use this config, choose the one you want and add this configuration to your `package.json`:
```

Comment:
This is more of a question and not an issue. Asking because it's saids "_choose the **one**_" but how about for hybrid WordPress/React projects, can we extend React and Gutenberg configs?

Reply:
The `@10up/eslint-config/wordpress` extends the react config so you can use that for hybrid set ups.

Additionally, if you have a decoupled project you could use just the react config on the front end and the WordPress config in the back end.

Reply:
And yes as with any eslint config you can extend and override rules in your project.